### ISSUE TITLE
THCudaCheck FAIL

### ISSUE THREAD

**Issue by chenzhanjie** [HKUST-Aerial-Robotics/Stereo-RCNN#25]
I run the project in pytorch 1.0 version, and then the result have been print. But there is a problem in the terminal:
THCudaCheck FAIL file=/opt/conda/conda-bld/pytorch_1544174967633/work/aten/src/THC/THCGeneral.cpp line=405 error=11 : invalid argument
I did not find the answer, so I hope that I can get some help at here. Thank you.

**Comment by PeiliangLi** [comment]
Have you installed cuda successfully?

**Comment by chenzhanjie** [comment]
> Have you installed cuda successfully?

Yes, i created an environment in Anaconda3, and then i installed the pytorch1.0 with cuda9.0 and python3.6.

**Comment by PeiliangLi** [comment]
What is your output of `nvidia-smi`, `nvcc --version`, and
`python`
`import torch`
`torch.__version__`?

**Comment by chenzhanjie** [comment]
> What is your output of `nvidia-smi`, `nvcc --version`, and
> `python`
> `import torch`
> `torch.__version__`?

these output you mentioned are right, but now i think the problem may be caused by network connection, I am in China. So just like you said,it is possible that i haven't installed it completely. But I do not find which part i missed yet. So I am trying to reinstall all the components once.

**Comment by immarioo** [comment]
i met the same error when i run python demo.py.
and my nvcc -V is v10.0(titan rtx), and my torch version is 1.0.0, torchvision is 0.2.2.
should the titan rtx install with cuda10.0? like conda install pytorch=1.0.0 cuda100 -c pytorch

**Comment by wanglaotou** [comment]
after run **conda install pytorch=1.0.0 cuda100 -c pytorch** and **conda install torchvision -c pytorch**, it worked for me, which it helps you : )

**Comment by immarioo** [comment]
after run conda install pytorch=1.0.0 cuda100 -c pytorch and conda install torchvision -c pytorch, it worked for me, wish it may help you : )
